qiskit/converters/circuit_to_instruction.py

```diff
@@ -41,6 +41,22 @@ def circuit_to_instruction(circuit, parameter_map=None):
         Instruction: an instruction equivalent to the action of the
             input circuit. Upon decomposition, this instruction will
             yield the components comprising the original circuit.
+
+    Example:
+        .. jupyter-execute::
```

Comment:
We probably don't need to execute this block since it doesn't really have any output. But it's fine